synapse/lib/stormtypes.py

```diff
@@ -1104,6 +1104,10 @@ def _get_mesg(mesg, **kwargs):
 
     @stormfunc(readonly=True)
     async def _print(self, mesg, **kwargs):
+        try:
+            mesg = await toprim(mesg)
```

Comment:
This does have some potentially backwards behaviors - for example:

```cli> storm [inet:ipv4=1.2.3.4] $lib.print($node) | spin
Executing query at 2021/05/11 15:22:03.611
...
Node{(('inet:ipv4', 16909060), {'iden': '20153b758f9d5eaaa38e4f4a65c36da797c3e59e549620fa7c4895e1a920991f', 'tags': {}, 'props': {'.created': 1620746523790, 'type': 'unicast'}, 'tagprops': defaultdict(<class 'dict'>, {}), 'nodedata': {}})}
complete. 0 nodes in 187 ms (0/sec).

vs 


cli> storm [inet:ipv4=1.2.3.4] $lib.print($node) | spin
Executing query at 2021/05/11 15:22:26.197
...
16909060
complete. 0 nodes in 78 ms (0/sec).
```

We also don't capture the list contents in the following example:

```
$list=$lib.list() $list.append(1) $list.append(2) 
$lib.print("List {list} has {len} elements!", list=$list, len=$lib.len($list))

> List <synapse.lib.stormtypes.List object at 0x7fa11503c100> has 2 elements!

```

I think the original task for for custom reprs for storm objects?